tox.ini

```diff
@@ -73,10 +73,10 @@ setenv =
 
 commands =
     # Run trial without coverage
-    test: trial --random=0 --logfile="{envlogdir}/trial.log" --temp-directory="{envlogdir}/trial.d" {posargs:{env:PY_MODULE}}
+    test: trial --random=0 --jobs=8 --logfile="{envlogdir}/trial.log" --temp-directory="{envlogdir}/trial.d" {posargs:{env:PY_MODULE}}
```

Comment:
why hardcode to 8 jobs ? ...  GitHub CI only have 2 CPUs 

I guess that we need `--jobs=auto` in trial :)

Reply:
auto would be best, yes. Will switch to 2

Reply:
See https://github.com/twisted/twisted/pull/1545#discussion_r591966698

Glyph raised a valid point. If the test are heave on IO you might want to run with `-j 4` :)
On Twisted,  I got a 10% improvement with -j4 over -j2 ... but with -j4 pypy tests were always failing

So -j8 might not be such a bad value for GitHub Actions :) and the speed improvement is amazing